packages/sdk/src/utils/sharing.ts

```diff
@@ -0,0 +1,74 @@
+import { gql, type GraphQLClient } from 'graphql-request';
```

Comment:
I'm not sure I want to to have these methods in the `utils/` folder. For me that should stay in the `sharing/` folder.

Maybe create a `dataProtector/sharing/utils` instead? With two files:
 - isCollectionOwner()
 - isProtectedDataInCollection()

Reply:
i agree